ee/session_recordings/queries/test/__snapshots__/test_session_recording_list_from_session_replay.ambr

```diff
@@ -0,0 +1,837 @@
+    AND s.min_first_timestamp >= '2024-02-22 00:00:00'
+    AND s.max_last_timestamp <= '2024-02-29 15:58:57'
+    AND s.distinct_id in
+      (select distinct_id
+       from
+         (SELECT distinct_id,
+                 argMax(person_id, version) as current_person_id
+          FROM person_distinct_id2 as pdi
+          INNER JOIN
+            (SELECT id
+             FROM person
+             WHERE team_id = 2
+               AND id IN
+                 (SELECT id
+                  FROM person
+                  WHERE team_id = 2
+                    AND (has(['bla'], "pmat_email")) )
+             GROUP BY id
+             HAVING max(is_deleted) = 0
+             AND (has(['bla'], argMax(person."pmat_email", version))) SETTINGS optimize_aggregation_in_order = 1) person ON person.id = pdi.person_id
+          WHERE team_id = 2
+          GROUP BY distinct_id
+          HAVING argMax(is_deleted, version) = 0) as session_persons_sub_query)
+  GROUP BY session_id
+  HAVING 1=1
+  ORDER BY start_time DESC
+  LIMIT 51
+  OFFSET 0
+  '''
+# ---
+# name: TestClickhouseSessionRecordingsListFromSessionReplay.test_poe_being_unavailable_we_fall_back_to_person_subquery
+  '''
+  
+  SELECT s.session_id,
+         any(s.team_id),
+         any(s.distinct_id),
+         min(s.min_first_timestamp) as start_time,
+         max(s.max_last_timestamp) as end_time,
+         dateDiff('SECOND', start_time, end_time) as duration,
+         argMinMerge(s.first_url) as first_url,
+         sum(s.click_count),
+         sum(s.keypress_count),
+         sum(s.mouse_activity_count),
+         sum(s.active_milliseconds)/1000 as active_seconds,
+         duration-active_seconds as inactive_seconds,
+         sum(s.console_log_count) as console_log_count,
+         sum(s.console_warn_count) as console_warn_count,
+         sum(s.console_error_count) as console_error_count
+  FROM session_replay_events s
+  WHERE s.team_id = %(team_id)s -- regardless of what other filters are applied
+   -- limit by storage TTL
+  
+    AND s.min_first_timestamp >= %(clamped_to_storage_ttl)s -- we can filter on the pre-aggregated timestamp columns
+   -- because any not-the-lowest min value is _more_ greater than the min value
+   -- and any not-the-highest max value is _less_ lower than the max value
+  
+    AND s.min_first_timestamp >= %(start_time)s
+    AND s.max_last_timestamp <= %(end_time)s
+    AND s.distinct_id in
+      (select distinct_id
+       from
+         (SELECT distinct_id,
+                 argMax(person_id, version) as current_person_id
+          FROM person_distinct_id2 as pdi
+          INNER JOIN
+            (SELECT id
+             FROM person
+             WHERE team_id = %(team_id)s
+               AND id IN
+                 (SELECT id
+                  FROM person
+                  WHERE team_id = %(team_id)s
+                    AND (has(%(vperson_filter_pre__0)s, replaceRegexpAll(JSONExtractRaw(properties, %(kperson_filter_pre__0)s), '^"|"$', ''))) )
```

Comment:
and with no materialized column and pOE off we join on person and extract the value from properties